tests/invalid_models_tests/test_models.py

```diff
@@ -1212,9 +1212,8 @@ def test_property_and_related_field_accessor_clash(self):
         class Model(models.Model):
             fk = models.ForeignKey('self', models.CASCADE)
 
-            @property
-            def fk_id(self):
-                pass
+        # Override related field accessor.
+        Model.fk_id = property(lambda self: 'ERROR')
```

Comment:
if you can't make this clash via `@property` any more, is the check still valid? Or at least could the error message and/or test model be changed?

Reply:
I've pushed a second commit removing this test. 

Not 100% convinced... 

We (@felixxm) discussed previously on the original PR. IIRC we said we'd leave it, and maybe adjust to add a check going the other way. (i.e. Cover _I tried to define the `pk_id` property, but it got overridden_ vs the previous _Because I'd defined the property the field descriptor wasn't set_.) 

I'd like to address that as a separate cleanup ticket. 

* This has been going on too long already. 
* And there are real issues being fixed here vs a system check/config error. 

🤔

Reply:
Okay fine by me, as long as the ticket exists :)

Reply:
Create follow-up at ticket-32847

(Will leave this conversation unresolved so folks maybe see it as they're browsing.)